assets/js/util/chart.js

```diff
@@ -73,3 +73,25 @@ export const isSingleSlice = ( report ) => {
 
 	return false;
 };
+
+/**
+ * Calculates difference between two values.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} currentValue  The difference which can be positive or negative.
+ * @param {number} previousValue Options to be passed to the ChangeArrow component.
```

Comment:
@kuasha420 these descriptions look irrelevant to what these parameters are. Can you adjust it?